feat: add tokenjs llm provider

Rawaa-Al-Kabbani opened the pull request:
Start using TokenJS as the LLM provider.

CGeorges commented on Close:
@Rawaa-Al-Kabbani was this file created by mistake?

CGeorges commented:
This looks great, can you update Readme to reflect we're using TokenJS for LLM integration and provide a link to supported LLM list?

Rawaa-Al-Kabbani commented:
> This looks great, can you update Readme to reflect we're using TokenJS for LLM integration and provide a link to supported LLM list? along with fixes for the linting issues reported by biome

Thank you! I fixed some bugs, updated the README and deleted Close file.
